tbx/static_src/sass/components/navigation/_sub-nav.scss

```diff
@@ -1,106 +1,235 @@
 @use "config" as *;
 @use "sass:color";
 
+/*
+Styles for the lower levels of the navitation
```

Comment:
```suggestion
Styles for the lower levels of the navigation
```

Reply:
Clearly my spelling yesterday was all over the place...